Solutions/Cisco Firepower EStreamer/Playbooks/CustomConnector/readme.md

```diff
@@ -52,8 +52,8 @@ Prior using this custom connector, it should be deployed in the Resource Group w
 <br>
 
```

Comment:
@v-amolpatil : why is this delete and add, it should be moved

Reply:
@v-rucdu  I am not sure why it is now showing as the add. I tried to revert this file to the previous folder and then move back to the solution folder but still it is showing as a add. Please refer PR commits here for more. Let me know if you have any other way/option to do this in order to close this comment.